open-craft/ansible-playbooks
# [BB-2527] Add a role for deploying a Matomo instance

**Testing instructions**:
* Deploy a Matomo instance using this role or alternatively test an already deployed instance (see the corresponding `ansible-secrets` PR)
